_posts/2021-02-08-ill-be-back.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+header_image: /img/posts/ill-be-back/header.jpg
+title: "I'll be back"
+author: Mauro Servienti
+synopsis: ""
+tags:
+- Distributed Systems
+- Timeouts
+- Long running
+- Sagas
+- Delayed messages
+---
+
+Time, as a concept, is both elementary and incredibly complex. Up to the point that physicians like Carlo Rovelli say that [it doesn't exist](https://youtu.be/vgsoI4ZUkUA). If you spend some time, ironically, time exists if you can spend it, diving into the theories' nuances, it's clear that time exists at a particular scale, it's not unique and ubiquitous, and it can be stretched and manipulated. And yes, at the quantum scale, it doesn't exist. Mind-blowing and fascinating.
```

Comment:
```suggestion
Time, as a concept, is both elementary and incredibly complex, to the point that physicists like Carlo Rovelli say that [it doesn't exist](https://youtu.be/vgsoI4ZUkUA). If you spend some time (ironically, time exists if you can spend it) diving into the nuances, it's clear that time exists at a particular scale, it's not unique and ubiquitous, and it can be stretched and manipulated. And yes, at the quantum scale, it doesn't exist. Mind-blowing and fascinating.
```